prime-router/src/testIntegration/kotlin/datatests/MappingInventoryTests.kt

```diff
@@ -0,0 +1,79 @@
+package gov.cdc.prime.router.datatests
+
+import gov.cdc.prime.router.Report
+import gov.cdc.prime.router.cli.tests.CompareData
+import org.junit.jupiter.api.Test
+
+class MappingInventoryTests {
+    fun shouldTranslateLosslessly(inputFile: String, outputFile: String): CompareData.Result {
+        val outputSchemaPath = "metadata/hl7_mapping/ORU_R01/ORU_R01-test"
+
+        val testConfig = TranslationTests.TestConfig(
+            inputFile, Report.Format.FHIR, "", outputFile,
+            Report.Format.HL7, outputSchemaPath, true, null, null, null
+        )
+        return TranslationTests().FileConversionTest(
+            testConfig
+        ).runTest()
+    }
+
+    @Test
+    fun `XTN use code home`() {
+        assert(
+            shouldTranslateLosslessly(
+                "mapping-inventory/xtn/xtn_use_code_home_2.fhir",
+                "mapping-inventory/xtn/xtn_use_code_home_2.hl7"
+            ).passed
+        )
+    }
+
+    @Test
+    fun `XTN use code mobile`() {
+        assert(
```

Comment:
@JessicaWNava I think Arnej was recommending that we comment the row in the inventory that we're asserting against?

Reply:
No, he was saying to put the row as part of the file name. I guess I can add a comment too.